src/core/Folder.ts

```diff
@@ -35,7 +34,7 @@ export class Folder implements IFolder {
 
   public legalDocuments: LegalDocument[];
 
-  public signatures: Signature[];
+  public signatures: ISignature[];
```

Comment:
Why did you change that? With this change, is it still possible to call `Signature#update`?

Reply:
You're right, I shouldn't